XenModel/Messages.resx

```diff
@@ -2020,6 +2020,18 @@ Apply update '{0}' to server '{1}'...
   <data name="AUTHORIZING_USER" xml:space="preserve">
     <value>Authorizing User</value>
   </data>
+  <data name="AUTOMATED_UPDATES_LICENSE_CHECK_DESCRIPTION" xml:space="preserve">
+    <value>License status for automated updates</value>
+  </data>
+  <data name="AUTOMATED_UPDATES_UNLICENSED_PROBLEM_MORE_INFO" xml:space="preserve">
+    <value>This problem may be caused by various reasons. Please consider one of the following solutions and try again:

```

Comment:
Some small improvements to portions of the text:
"This problem may have various causes ... "
" ... Ensure automated updates are not restricted on any hosts in the pool ..." (no need to mention feature flags)
"... [XenCenter] will upgrade the selected servers to the new version without applying any updates. " (it's the application applying the updates, not the servers)